src/main/java/green/citibike/aws/Response.java

```diff
@@ -0,0 +1,41 @@
+package green.citibike.aws;
+
+import green.citibike.json.StationInfo;
+import green.citibike.json.StationStatus;
+
+public class Response {
+    private Coordinate from;
+    private StationInfo start;
+    private Coordinate to;
+    private StationInfo end;
+
+    public Response(Coordinate from, StationInfo start, Coordinate to, StationInfo end) {
+        this.from = from;
+        this.start = start;
+        this.to = to;
+        this.end = end;
+    }
+
+    public Response(StationStatus from, StationStatus to) {
+        this.from = new Coordinate(from.getLat(), from.getLon());
+        this.start = from.getStationInfo();
+        this.to = new Coordinate(to.getLat(), to.getLon());
+        this.end = to.getStationInfo();
```

Comment:
`this.from` and `this.to` are the original coordiantes that were passed to this lambda in the request.

Reply:
So they're still stored? How?

Reply:
You need to save it